Remotion/Web/Core/UI/Controls/ListMenuImplementation/Rendering/ListMenuRenderer.cs

```diff
@@ -323,8 +325,13 @@ private void AppendListMenuItem (ListMenuRenderingContext renderingContext, Stri
         if (!string.IsNullOrEmpty(menuItem.ItemID))
           diagnosticMetadataDictionary.Add(DiagnosticMetadataAttributes.ItemID, menuItem.ItemID);
 
-        if (!string.IsNullOrEmpty(menuItem.Text))
-          diagnosticMetadataDictionary.Add(DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags(menuItem.Text));
+        if (!menuItem.Text.IsEmpty)
+        {
+          var contentAttributeValue = menuItem.Text.Type == WebStringType.Encoded
+              ? HtmlUtility.StripHtmlTags (menuItem.Text.GetValue())
+              : menuItem.Text.GetValue();
```

Comment:
What do you think, should we do an overload for StripHtmlTags that accepts WebString and wraps this semantic? It feels like a good idea given we already have this web-test-helper method anyway.

Reply:
I think it is a good idea, that also solves the problem for future tasks because the code now compiles without problems :D

Reply:
> hat also solves the problem for future tasks because the code now compiles without problems :D
Am not sure how to interpret the "compiles without problems" part. Was this about doing this change "now vs later" or something else?

Reply:
Other code that uses the `Content` diagnostic metadata attribute already uses the `HttpUtility.StripHtmlTags` but with the `string` overload. If a future task changes the type of a property to `WebString` it will just compile without change because the `WebString` overload of `HttpUtility.StripHtmlTags` is used

Reply:
Could you pelase create a ticket for this refactoring? thanks!

Reply:
https://re-motion.atlassian.net/browse/RM-8170